CampaignAutomation/Tests/CampaignAutomationImplementingTests.swift

```diff
@@ -104,6 +104,8 @@ class CampaignAutomationImplementingTests: XCTestCase {
                 return XCTFail("Test failed")
             }
             XCTAssertEqual((error as? DummyError), DummyError.failure)
+            // ensure that cache was accessed after server query failure
+            XCTAssertEqual(self.graphQLManager.numberOfQueriesCalled, 2)
```

Comment:
Bitrise failure was due to this line, and the error message is:
![screenshot0328](https://user-images.githubusercontent.com/77625840/160373609-0dd92745-5535-4e77-a6f9-8ee944fe161c.png)
https://app.bitrise.io/build/0ac41e9e-e86a-4a4b-92f4-129ee1049ce7